### PR TITLE
completed

### PR DESCRIPTION
**Pull request by bug-smith**
Screenshot 2023-09-07 at 4 09 32 PM
completed

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h1>Choose your character!</h1>
+      </div>
+      <div class="row">
+        <img class="character column-third" src="images/link.png" alt="Link" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: This `img` should be inside a `div` that has the class `column-third`. The `img` shouldn't have the column class

File: css-layout-classes/index.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+            critics and players, with some critics calling it the best
+            installment in the series. it recevied praise for its large amount
+            of content and fine-tuning of existing Smash gameplay elements,
+            although its online mode was widely criticized.
+          </p>
+        </div>
+        <img
+          class="character column-third"
+          src="images/yoshi.png"
+          alt="Yoshi" />
```
Comment (shawnkost) [inline css-layout-classes/index.html]: Same note as above ^

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Bug!

**Comment by bug-smith** [comment]
completed.
Screenshot 2023-09-08 at 10 19 53 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**